docs/server/source/quickstart.md

```diff
@@ -8,7 +8,7 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 B. Run MongoDB. Open a Terminal and run the command:
 ```text
-$ mongod --replSet=bigchain-rs
+$ sudo mongod --replSet=bigchain-rs
```

Comment:
Did anyone complain about this?
I don't think that running mongod with sudo is a good solution